src/pages/GoverningBody/CreateGoverningBody.tsx

```diff
@@ -40,14 +41,20 @@ import {
   failCreateAction,
   failUpdateAction,
 } from "../../components/Notifications/Messages"
-import { descriptionValidation } from "../../models/GllobalValidations/DescriptionValidation";
+import { descriptionValidation, sameNameValidator} from "../../models/GllobalValidations/DescriptionValidation";
 
 const CreateGoverningBody = () => {
   const { id } = useParams();
   const history = useHistory();
 
   const [loading, setLoading] = useState(false);
   const [governingBody, setGoverningBody] = useState<GoverningBodyProfile>(new GoverningBodyProfile());
+  const [governingBodyNames, setGoverningBodyNames] = useState<any[] | undefined>();
```

Comment:
do we have a specific model for `GoverningBodyNames`?
if yes, we should use it instead of `any[]`
if no, we should create and use it